engines/python/setup/djl_python/tests/test_rolling_batch.py

```diff
@@ -28,18 +28,14 @@ def test_json_fmt(self):
         req2 = Request(req_input2)
         for req in [req1, req2]:
             req.set_next_token(Token(244, "He", -0.334532))
-            print(req.get_next_token(), end='')
-            assert req.get_next_token() == '{"generated_text": "He'
-            req.reset_next_token()
+            req.get_next_token()
             req.set_next_token(Token(576, "llo", -0.123123))
-            print(req.get_next_token(), end='')
-            assert req.get_next_token() == 'llo'
-            req.reset_next_token()
+            req.get_next_token()
             req.set_next_token(Token(4558, " world", -0.567854), True,
                                'length')
             print(req.get_next_token(), end='')
-            assert req.get_next_token() == ' world"}'
-            req.reset_next_token()
+            assert req.get_next_token() == json.dumps(
```

Comment:
self.assertEquals

Reply:
let me take this up as a separate change. We don't have consistent usage of assert/self.assert<Condition> across our tests, so i can fix on that across our tests